bait-files/src/main/webapp/userHistory.js

```diff
@@ -0,0 +1,52 @@
+const url =
+    'https://step-bait-project-2020.uc.r.appspot.com/review.html?reviewId=';
+
+function getCompletedReviews() {
+  fetch('/userHistory?is-pending-review=false')
+      .then(response => response.json())
+      .then((reviews) => {
+        for (review of reviews) {
+          addReview(review, document.getElementById('completed-table'));
+        }
+      });
+}
+
+function getPendingReviews() {
+  fetch('/userHistory?is-pending-review=true')
+      .then(response => response.json())
+      .then((reviews) => {
+        for (review of reviews) {
+          addReview(review, document.getElementById('pending-table'));
+        }
+      });
+}
+
+function addReview(review, tableToAddTo) {
+  const queue = tableToAddTo;
+  const row = document.createElement('div');
+  row.setAttribute('class', 'table-body-row');
+
+  const timestamp = document.createElement('div');
```

Comment:
maybe write a helper function since creating the div element and setting the attribute is continually used in this block of code

Reply:
Created a helper method